DuckDuckGo/AppDelegate.swift

```diff
@@ -352,6 +359,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 extension AppDelegate: BlankSnapshotViewRecoveringDelegate {
     
     func recoverFromPresenting(controller: BlankSnapshotViewController) {
+        if overlayWindow == nil {
+            tryToObtainOverlayWindow()
+        }
```

Comment:
This is additional level of "make sure we have an overlay" as @samsymons found that we may run into situations where `overlayWindow` is nil, but overlay is still there. Obtaining it and setting hidden property directly is supposed to help.